### ISSUE TITLE
Contexts not expanding

### ISSUE THREAD

**Issue by stueja** [todotxt/todo.txt-cli#255]
**Do you want to request a *feature* or report a *bug*?**
A bug.

**What is the current behavior?**
Entering "@" and hitting "tab" does not expand the contexts.

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**
Enter `t add +project @` and hit tab.

**What is the expected behavior?**
Like with `+`, see a list of contexts.

**Which versions todo.sh are you using?**
> Run `todo.sh -V`
DEV_VERSION

**Which Operating System are you using?**
Arch Linux

**Which version of bash are you using?**
> Run `bash --version`
4.4.23

Note: I sent this to the mailing list, no answer so far.

**Comment by pabgan** [comment]
I guess you did, but just in case: Have you added `complete -F _todo t` to `todo_completion` file in todo-txt directory?

**Comment by stueja** [comment]
Hey @pabgan , yes, I have. Everything else is expanding properly. Only the contexts are not.

**Comment by inkarkat** [comment]
That's strange; if `+project` completion is working, `@context` should, too. Do you actually have contexts in your `todo.txt`; do you get output from 

    todo.sh command listcon

**Comment by stueja** [comment]
Yes, I do have contexts, and `todo.sh command listcon` outputs them.

More interestingly, either I made a mistake in my original post, or it now _partly_ works:
`todo.sh add @<tab>` outputs, indeed the list of contexts. E. g.:
```
[stueja@host ~]$ todo.sh add @
@daily       @calendar    @linux       @monthly     @online
```
However, _expansion_ does not work:
`todo.sh add @lin<tab>` will not expand to `@linux`. Whereas `todo.sh add +pro<tab>` will well expand to `+project`

**Comment by sjdennis** [comment]
I am seeing command expansion working, but neither projects nor contexts working. For instance:
t li<tab> expands to t list.
But t add +ca<tab> will not expand to +camp, nor will @ca<tab> expand.

t listproj and t listcon both work as expected, returning lists of projects and contexts in todo.txt.

Using version 2.11.0 on Ubuntu 16.04
GNU bash, version 4.3.48(1)-release (x86_64-pc-linux-gnu)

**Comment by l00kanio** [comment]
I had the same issue as @stueja wrote. Check if there is  'shopt -s hostname' in your .bashrc file.
If yes => comment it out and have fully functional autocompletion with todotxt :-)

**Comment by sjdennis** [comment]
I have no such line in my .bashrc unfortunately, so that doesn't help in my case. Thanks for the suggestion though!
